service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -275,15 +278,29 @@ void shouldGenerateCorrectFileNameGivenOrderType(GeneratedOrderType type,
     @ParameterizedTest
     @MethodSource("docmosisDataGenerationSource")
     void shouldCreateExpectedMapWhenGivenPopulatedCaseData(GeneratedOrderType orderType,
-                                                           GeneratedOrderSubtype subtype) {
+                                                           GeneratedOrderSubtype subtype) throws IOException {
         LocalDateTime now = time.now();
         CaseData caseData = createPopulatedCaseData(orderType, subtype, now.toLocalDate());
 
         Map<String, Object> expectedMap = createExpectedDocmosisData(orderType, subtype, now);
-        Map<String, Object> templateData = service.getOrderTemplateData(caseData);
+        Map<String, Object> templateData = service.getOrderTemplateData(caseData, SEALED);
 
-        assertThat(templateData).isEqualTo(expectedMap);
+        assertThat(templateData).containsAllEntriesOf(expectedMap);
+        assertThat(templateData).containsKey("courtseal");
```

Comment:
should this assertion not be in expectedMap?

Reply:
so the method expectedMap is huuuge switch statement based on what order.. and courtseal will only exist in it if it's sealed and draftbackground will only exist if draft.. rather than created two of the same methods expectedMapCourtSeal expectedMapDraftBackground I thought i'd do it this way, happy to change i just thought this would save adding two big methods..